code/datums/gamemode/role/catbeast.dm

```diff
@@ -38,7 +38,7 @@
 
 var/list/catbeast_names = list("Meowth","Fluffy","Subject 246","Experiment 35a","Nyanners","Thing From Below","Airlock Scratcher","Flees-Like-Fleas",
 						"Lurks-In-Shadows","Eartha Kitt","Target Practice","Fresh Meat","Ca'thulu","Furry Fury","Vore-Strikes-Back","Killing Machine","Uncle Tom",
-						"Nine Lives", "Bad Luck", "Siamese Sam", "Tom Tabby", "Hairball", "Throws-Dice-Poorly", "Wizard Apprentice", "Lynch Lynx", "Felix", "Niggerman")
+						"Nine Lives", "Bad Luck", "Siamese Sam", "Tom Tabby", "Hairball", "Throws-Dice-Poorly", "Wizard Apprentice", "Lynch Lynx", "Felix")
```

Comment:
Someone suggested "Lovecraft's Cat" could replace this, which I think was funny. I wrote every catbeast name except this one in question (which Adam submitted)

Reply:
i'll have you know that i came up with nyanners

Reply:
Based Idea, did it there.